pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -9,19 +9,24 @@
 
 
 @kwargs_to_strings(region="sequence")
-def load_earth_magnetic_anomaly(resolution="01d", region=None, registration=None):
+def load_earth_magnetic_anomaly(
+    resolution="01d", region=None, registration=None, mag4km=False
+):
     r"""
     Load an Earth magnetic anomaly grid in various resolutions.
 
     The grids are downloaded to a user data directory
-    (usually ``~/.gmt/server/earth/earth_mag/``) the first time you invoke
+    (usually ``~/.gmt/server/earth/earth_mag/`` or
+    ``~/.gmt/server/earth/earth_mag4km/``) the first time you invoke
     this function. Afterwards, it will load the grid from the data directory.
     So you'll need an internet connection the first time around.
 
     These grids can also be accessed by passing in the file name
-    **@earth_mag**\_\ *res*\[_\ *reg*] to any grid plotting/processing
-    function. *res* is the grid resolution (see below), and *reg* is grid
-    registration type (**p** for pixel registration or **g** for gridline
+    **@**\ *earth_mag_type*\_\ *res*\[_\ *reg*] to any grid plotting/processing
+    function. *earth_mag_type* is the GMT name
+    for the dataset. The available options are **earth_mag** and
+    **earth_mag4km**\. *res* is the grid resolution (see below), and *reg* is
```

Comment:
Is the `\` required?
```suggestion
    **earth_mag4km**. *res* is the grid resolution (see below), and *reg* is
```

Reply:
No; just tested it and it is not required.